checkstyle/suppressions.xml

```diff
@@ -37,6 +37,7 @@
 
     <!-- core -->
     <suppress checks="NPathComplexity" files="(ClusterTestExtensions|KafkaApisBuilder|SharePartition|SharePartitionManager).java"/>
+    <suppress checks="ParameterNumber" files="SharePartitionManager.java"/>
```

Comment:
Can we explicitly have this on top of method where the problem occurs instead of suppressions file?